Nodejs/Product/TestAdapter/TestFrameworks/Angular/vsKarmaReporter.js

```diff
@@ -64,6 +64,7 @@ const vsKarmaReporter = function (baseReporterDecorator, config, logger, emitter
             filepath: fullFilePath,
             line: result.fileLocation.line,
             column: result.fileLocation.column,
+            configPath: project.angularConfigPath,
```

Comment:
This is the (only?) place where the new configPath that gets threaded through everywhere is actually specified?  Where is it consumed?

Reply:
Yes, this is where we specify it.
During discovery, this will be consumed by the VS Test Adapter in `TestDiscovererWorker.cs:81`.
During execution this will be a property and used here: `TestExecutorWorker.cs:387`.

Hope it makes sense.